test/openreevoo_test.exs

```diff
@@ -1,7 +1,40 @@
 defmodule OpenreevooTest do
   use ExUnit.Case
 
-  test "the truth" do
-    assert 1 + 1 == 2
+  @tag :skip
+  test "The application asks the user to log in if they aren't already" do
   end
```

Comment:
If you're using Elixir 1.1 you can do this:

```elixir
test "The application asks the user to log in if they aren't already"
```

And then you don't need to tag with `:skip`